### ISSUE TITLE
[FE] My VA: Health care update to "Schedule an appointment" related to MHV/Cerner split

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#7286]
## User story

As a person who receives medical care from VA facilities in Columbus, OH, I will need to start going to a different VA system than MHV on April 10 to schedule appointments. 

## Background

Design in #5733

The VA is moving to a new health care system — Cerner — over the course of the next couple of years. The very first part of this move launches in April, where veterans that go to the Chalmers P. Wylie Ambulatory Care Center will need to start using a new tool — My VA Health — to schedule appointments. All other veterans will still go to MyHealtheVet to schedule appointments.

As a result, we need to show an alert on the My VA dashboard that lets users know about this switch.

### Design

![Dashboard-Columbus notification](https://user-images.githubusercontent.com/34068740/77588392-21ea4980-6ec0-11ea-910a-e8a5c771731c.png)

### Copy

**Our records show that you’re registered at Chalmers P. Wylie Ambulatory Care Center. Starting April 10, 2020, providers at this center are using My VA Health to manage appointments.**

You may need to sign in again to use VA health appointment tools. If you do, sign in with the same account you used to sign in on VA.gov. You may need to disable your browser’s pop-up block so these tools can open.

**Manage appointments with a Chalmers P. Wylie provider**
[Button] Go to My VA Health

**Manage appointments at all other VA medical centers**
[Button] Go to VA appointments

### Specs

**General**

- This will go in the "Schedule an appointment" section of the dashboard **for Cerner users only** (more on how to identify them below). For users who are *not* Cerner patients, they can still see what we have now in that section.
- Use the yellow alert state.

**Identifying the right users**

When the user is logged into va.gov, from the FE you can make a `GET /v0/user` request to get information about the user.  Under the path `data["attributes"]["va_profile"]["is_cerner_patient"] `there will be a boolean that indicates whether or not they are a cerner patient.

**Button destinations**

- Go to My VA Health:
  - **Staging URL**: https://ehrm-va-test.patientportal.us.healtheintent.com/
  - **Production**: https://patientportal.myhealth.va.gov/
- Go to VA appointments: 
  - **Staging URL**: https://staging.va.gov/health-care/schedule-view-va-appointments/
  - **Production**: https://www.va.gov/health-care/schedule-view-va-appointments/
## Tasks

- [ ] Build the alert

## Acceptance Criteria

- [ ] Alert is ready to go when the Cerner launch happens

**Comment by Samara-Strauss** [comment]
@ambikaroos @bridgethapner @hinzed1127 

Just double checking — the only facility affected right now is Chalmers P. Wylie Ambulatory Care Center, correct? @bradl3yC wanted to confirm that the facility name is not being pulled from the API. I was 99.9% sure that it was not, but thought confirming again couldn't hurt.

**Comment by erikphansen** [comment]
@Samara-Strauss where did you get the instructions to check `data["attributes"]["va_profile"]["is_cerner_patient"]`? We could use a little more info about when that will be set to `true`. Will it be set to `true` as long as they are a patient in _a_ Cerner facility? Or will it only be set to `true` if they are patients in _only_ Cerner facilities?

**Comment by Samara-Strauss** [comment]
@erikphansen I got that from @ambikaroos. She and @hinzed1127 would be the best people to direct questions to about this.

**Comment by erikphansen** [comment]
@ambikaroos or @hinzed1127 can either of you clarify that this new CTA will be shown if:
- a Vet is a patient in _at least one_ Cerner facility?
- or, a Vet is a patient in _only_ Cerner facilities?

**Comment by ambikaroos** [comment]
@erikphansen it's the former - this CTA should be shown if a Vet is a patient in _at least one_ Cerner facility.
cc @bridgethapner

**Comment by hinzed1127** [comment]
@erikphansen a little additional context on the implementation end - here's the [changes](https://github.com/department-of-veterans-affairs/vets-api/pull/3947) in `vets-api` that does that check.

**Comment by bradl3yC** [comment]
@Samara-Strauss @mshea0606 this is ready for validation

**Comment by Samara-Strauss** [comment]
@bradl3yC thanks! I don't think we have a staging user for this *yet* (on a different team to get that coordinated), so we'll have to hold off until then.

**Comment by Samara-Strauss** [comment]
@bradl3yC I just got word that we need to put this on hold. The Cerner launch has been delayed.

@lenaecb @ambikaroos @bridgethapner @hinzed1127 Could one of you let us know the following:

- Is there a new target date for launch?
- Do any of the link locations need to change? This is what we have now:
  - **Go to My VA Health**:
    - Staging URL: https://ehrm-va-test.patientportal.us.healtheintent.com/
    - Production: https://patientportal.myhealth.va.gov/
  - **Go to VA appointments**:
    - Staging URL: https://staging.va.gov/health-care/schedule-view-va-appointments/
    - Production: https://www.va.gov/health-care/schedule-view-va-appointments/
- Do we need to make any other changes to the work outlined at the top of this ticket?

**Comment by ambikaroos** [comment]
@Samara-Strauss re: the target date - the most recent info we have from @steve-gov as of yesterday afternoon is that we still need to aim for 4/10 (though this may change pending confirmation). I'm not aware of this changing the staging or production links for My VA Health, but can update you if I learn otherwise! cc @marriej

**Comment by Samara-Strauss** [comment]
@ambikaroos OK, then we'll proceed as we have been. We're relying on y'all to let us know if any of the changes around launch affect our work, so please keep us posted and let us know if we need to update anything.

**Comment by lenaecb** [comment]
@Samara-Strauss - tagged you in a slack convo about these links and how we're handling them for static VA.gov scheduling page. https://dsva.slack.com/archives/GM5JJ5388/p1585754424007600

TLDR; VAOS is not rolled out to 100%/rollout currently paused. 2nd CTA should point to MHV, however, when we start VAOS rollout again soon we'll want that 2nd CTA to point to new VAOS if that user is part of the user % increase. That conditional logic is already in place (or should persist based on our understanding) for this use case - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/cta-widget/helpers.js#L112 (MHV link is built for the CTA widget in that file)

The same conditions for those CTAs should also apply for MY VA tab. Let me know if you have additional questions about this.

**Comment by Samara-Strauss** [comment]
@lenaecb ok, understood that we should still point patients to MHV instead of the new VAOS tool. However, right now, the "Manage appointments at all other VA medical centers" link is going to filter people through https://www.va.gov/health-care/schedule-view-va-appointments/. Since this page currently points people to MHV, does that work? Or should we still change the link location for "Manage appointments at all other VA medical centers"?

![Dashboard-Columbus notification](https://user-images.githubusercontent.com/34068740/77588392-21ea4980-6ec0-11ea-910a-e8a5c771731c.png)

**Comment by lenaecb** [comment]
@Samara-Strauss - ah yes, I see. I guess I was just thinking we'd treat that alert box with the same functionality that we're using on the static appointment page that you are currently pointing to now from My VA tab (https://www.va.gov/health-care/schedule-view-va-appointments/) to cut down on user clicks. But makes sense to continue what we're already doing.

**Comment by Samara-Strauss** [comment]
Awesome, sounds good, @lenaecb!

I think this should be good to go besides needing a staging user to verify the functionality. I know there have been various conversations about that on Slack and that a staging user doesn't yet exist. Do you know if there's actually a plan to set that up, or are we relying on developers to verify functionality locally?

**Comment by lenaecb** [comment]
@Samara-Strauss - we also are trying to find a testing user for our Cerner work as well. I have been tracking your slacks about this, but also not sure where we are on getting that user. I know our team tested locally but I would reallllly like to test with a test user.

**Comment by Samara-Strauss** [comment]
OK, thanks @lenaecb. Keep us posted if you get something. 

Otherwise, @bradl3yC — I think local verification that this works will have to suffice for validating. We can circle back if we're eventually able to get a staging user.

**Comment by Samara-Strauss** [comment]
PR here: https://github.com/department-of-veterans-affairs/vets-website/pull/12049 which launched this to staging and prod. I'm having @bradl3yC remove this from prod, though no one should see it right now. Once Cerner is ready to launch, we can re-launch to prod.

@bradl3yC can you drop the PR here for removing this from prod when the PR is ready?

**Comment by bradl3yC** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/12134

**Comment by DanielleThierryUSDSVA** [comment]
Hi @Samara-Strauss. Dropping in the live site updated content. Let me know if you have any questions. Thanks!

![image](https://user-images.githubusercontent.com/62957278/91207187-63348e80-e6d6-11ea-8614-2c4a7737ec56.png)
